spec/acceptance/running_strings_generate_spec.rb

```diff
@@ -0,0 +1,62 @@
+require 'spec_helper_acceptance'
+include PuppetLitmus # rubocop:disable Style/MixinUsage This is fine
+
+describe 'Generating module documentation using generate action' do
+  before :all do
+    # TODO: Linux only
+    @sut_work_dir = PuppetLitmus::Serverspec.run_shell("pwd").stdout.chomp
+
+    test_module_path = sut_module_path(/Module test/)
+    PuppetLitmus::Serverspec.run_shell("puppet strings generate \"#{test_module_path}/**/*.{rb,pp}\"")
+  end
+
+  def expect_file_contain(path, expected_contents)
```

Comment:
it would be good to move this helper function out to spec_helper_acceptance_local.rb

Reply:
Yeah.  If it's needed outside of this file sure. Currently it's not.

Personally, I prefer to keep "magic" as close to the tests as possible to stop people chasing over different files e.g. this project used `/spec/acceptance/lib/*` and `/spec/spec_acceptance_helper.rb` to hold it's helper code.